presto-spark-launcher/src/main/java/com/facebook/presto/spark/launcher/PrestoSparkClientOptions.java

```diff
@@ -34,4 +34,7 @@
 
     @Option(name = "--schema", title = "schema", description = "Default schema")
     public String schema;
+
+    @Option(name = "--session-property-config", title = "file", description = "session-property-configuration.properties path")
```

Comment:
can we just call it "--session-property"?

Reply:
@kewang1024 recommend "--session-property-config"

Reply:
"--session-property" sounds simpler